### PR TITLE
Make file size uploads configurable via env vars

### PR DESCRIPTION
**Pull request by DistroByte**
The information contained in an actual budget is now much more detailed than it was previously. Doubling this file size should prevent errors

### CODE REVIEW

**Comment by Scot-Survivor** [comment]
Would it not be better to have these configurable via environment variable(s)?

**Comment by DistroByte** [comment]
I've added them as env vars now

File: src/app.js
Comment (MatissJanis) [inline src/app.js:30]:
```suggestion
app.use(bodyParser.json({ limit: `${config.upload.fileSizeLimitMB}mb` }));
```

File: src/load-config.js
Comment (DistroByte) [reply src/load-config.js:97]: Okay, so a person can set only one of these vars, rather than all 3?

**Comment by DistroByte** [comment]
Let's squash this PR if it gets merged, I am quite happy for people to never come across this atrocity ever again

File: src/load-config.js
Comment (MatissJanis) [inline src/load-config.js:110]: Actually, I think there is one more bug.

If the use has not provided these env vars and he is not using a config file: the `fileSizeLimitMB` vars will not be set. Which means `app.js` will break.

File: src/load-config.js
Comment (MatissJanis) [inline src/load-config.js:107]: Some of these vars are still using the old name.

**Review by MatissJanis — CHANGES_REQUESTED** [review]
Almost there.. :) 

Would you mind also doing a test run after you're happy with the changes? Just to verify that it still works as expected.

File: src/load-config.js
Comment (DistroByte) [reply src/load-config.js:107]: Good catch, resolved

File: src/load-config.js
Comment (DistroByte) [reply src/load-config.js]: I've added a section to `defaultConfig` for `upload`. That should resolve this

**Comment by DistroByte** [comment]
> Almost there.. :)
> 
> Would you mind also doing a test run after you're happy with the changes? Just to verify that it still works as expected.

Sure thing, is there an image I can pull or am I verifying this outside of docker?

**Comment by DistroByte** [comment]
I can confirm it works as expected with no configuration (defaults), and it also works with any combination of env vars

**Comment by MatissJanis** [comment]
Doesn't work for me when using env vars.

**Comment by DistroByte** [comment]
@MatissJanis this should be good to go now. Please review below examples, I can write some tests for this also if needs be

```bash
$ ACTUAL_UPLOAD_FILE_SIZE_LIMIT_MB=500 DEBUG=actual:config node app # set one, replicate to all upload limits
  actual:config project root: '/mnt/c/Users/james/Documents/GitHub/actual-server-1' +0ms
.
.
  actual:config using file sync limit 500mb +0ms
  actual:config using sync encrypted file limit 500mb +0ms
  actual:config using file limit 500mb +1ms
```

```bash
$ ACTUAL_UPLOAD_FILE_SYNC_SIZE_LIMIT_MB=500 DEBUG=actual:config node app # set one, rest remain default
.
.
  actual:config using file sync limit 500mb +0ms
  actual:config using sync encrypted file limit 50mb +1ms
  actual:config using file limit 20mb +0ms
```

```bash
$ ACTUAL_UPLOAD_SYNC_ENCRYPTED_FILE_SYNC_SIZE_LIMIT_MB=500 DEBUG=actual:config node app # set one, rest remain default
.
.
  actual:config using file sync limit 20mb +0ms
  actual:config using sync encrypted file limit 500mb +0ms
  actual:config using file limit 20mb +0ms
```

```bash
$ ACTUAL_UPLOAD_SYNC_ENCRYPTED_FILE_SYNC_SIZE_LIMIT_MB=500 ACTUAL_UPLOAD_FILE_SYNC_SIZE_LIMIT_MB=300 DEBUG=actual:config node app # set two, overall file limit remains at default
.
.
  actual:config using file sync limit 300mb +1ms
  actual:config using sync encrypted file limit 500mb +0ms
  actual:config using file limit 20mb +0ms
```

```bash
$ ACTUAL_UPLOAD_SYNC_ENCRYPTED_FILE_SYNC_SIZE_LIMIT_MB=500 ACTUAL_UPLOAD_FILE_SIZE_LIMIT_MB=100 DEBUG=actual:config node app
.
.
  actual:config using file sync limit 100mb +1ms # set by ACTUAL_UPLOAD_FILE_SIZE_LIMIT_MB
  actual:config using sync encrypted file limit 500mb +0ms
  actual:config using file limit 100mb +0ms
```

```bash
$ ACTUAL_UPLOAD_SYNC_ENCRYPTED_FILE_SYNC_SIZE_LIMIT_MB=500 ACTUAL_UPLOAD_FILE_SIZE_LIMIT_MB=100 ACTUAL_UPLOAD_FILE_SYNC_SIZE_LIMIT_MB=230 DEBUG=actual:config node app
.
.
  actual:config using file sync limit 230mb +0ms
  actual:config using sync encrypted file limit 500mb +0ms
  actual:config using file limit 100mb +0ms
```

**Review by MatissJanis — APPROVED** [review]
Looks good to me! Thanks for your contribution!

(we are currently in merge freeze until the next release which means only bugfixes and low-risk changes will be merged; I will merge this PR once we've opened up merges again)

**Pull request merged**
